components/threading/common/button.scss

```diff
@@ -0,0 +1,86 @@
+.Button {
+    padding: 4.5px 10px;
+    border-radius: 4px;
+    position: relative;
+
+
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: 600;
+
+    &:hover,
+    &:active {
+        text-decoration: none;
+    }
+
+    &.is-active {
+        color: rgba(var(--button-bg-rgb), 1);
+    }
+
+
+    &.Button___transparent {
+        background: transparent;
+        border: none;
+
+        &:hover,
+        &:focus {
+            background: rgba(var(--center-channel-color-rgb), .08);
+        }
+        &.is-active {
+            background: rgba(var(--button-bg-rgb), .08);
+            &:focus {
+                position: relative;
+                &:after {
+                    @extend %synthetic-block;
+                    border-radius: 4px;
+                    border: 2px solid rgba(var(--button-bg-rgb), 1);
+                }
+            }
+        }
+    }
+
+    .Icon svg {
+        width: 12px;
+        height: 10px;
+    }
+    .Button_prepended {
+        margin-right: .75rem;
+    }
+    .Button_appended {
+        margin-left: .75rem;
+    }
+    .dot {
+        position: absolute;
+        width: 4px; height: 4px;
+        right: 4px; top: 3px;
+        border-radius: 50%;
+
+        background: rgba(var(--sidebar-text-active-border-rgb), 1);
+    }
+}
+
+.Separated,
+.Button:not(:hover):not(:focus):not(.is-active) {
+    & + .Separated,
+    & + .Button:not(:hover):not(:focus):not(.is-active) {
+        position: relative;
+        &::before {
+            width: 1px;
+            height: 15px;
+            position: absolute;
+            left: -.5px;
+            background: rgba(var(--center-channel-color-rgb), .16);
+            content: '';
+            display: inline-block;
+        }
+    }
+}
```

Comment:
This is what creates the separator when two `.Button`s or `.Separated`s are next to each other.
![2020-09-30 16 20 46](https://user-images.githubusercontent.com/11724372/94741055-301a9600-0339-11eb-9ef0-30e59e302e2f.gif)